terraform/fec_rds.tf

```diff
@@ -93,13 +93,21 @@ resource "aws_db_instance" "rds_production" {
   publicly_accessible = true
   storage_encrypted = true
   multi_az = true
+  auto_minor_version_upgrade = true
+  storage_type = "io1"
+  identifier = "fec-govcloud-prod"
```

Comment:
The goal of terraform is to avoid having to use the console most of the time, and to capture all changes in configuration. Terraform lets you create an rds instance from a snapshot identifier, so I think you'd want to:

* Change the configuration to allow destroy, *and* to set the `snapshot_identifier` attributes of the instances you want to restore
* Apply changes via terraform (this should happen automatically on merge, but you might need to coordinate if there's a config error)
* Change the configuration to prevent destroy